src/definition/users/IUserCreation.ts

```diff
@@ -0,0 +1,14 @@
+export interface IUserCreation {
```

Comment:
@lolimay this looks like a duplicate of `IUser` 🤔 Is there a technical issue preventing us from using `IUser` during user creation methods?

Reply:
Upendra and @graywolf336 has discussed it before: https://github.com/RocketChat/Rocket.Chat.Apps-engine/pull/133#discussion_r332496728 What's your idea on it?

Reply:
I think we can replace this with `Partial<IUser>`; this makes all the fields on the `IUser` interface, which solves the problem discussed in that comment.

What do you think?

Reply:
Changed